src/components/NostoProduct.ts

```diff
@@ -62,6 +62,16 @@ function initProduct(element: NostoProduct) {
     element.selectedSkuId = selectedSkuId
     element.skuSelected = !!selectedSkuId
   })
+  store.listen("skuImage", image => {
+    if (image) {
+      element.style.setProperty("--ns-img", `url(${image})`)
+    }
+  })
+  store.listen("skuAltImage", altImage => {
+    if (altImage) {
```

Comment:
drop, since we can assume that this event will only be submitted when a valid skuAltImage has been set